app/src/main/java/com/globant/harrypotterapp/activity/CharacterDetailActivity.kt

```diff
@@ -83,12 +88,18 @@ class CharacterDetailActivity : AppCompatActivity() {
     }
 
     private fun showCharacterDetailError(message: String?) {
-        binding.characterDetailActivityLoader.visibility = View.GONE
-        Toast.makeText(this, message, Toast.LENGTH_SHORT).show()
+        with(binding) {
+            characterDetailActivityLoader.visibility = View.GONE
+            characterDetailActivityToolbar.title = getString(R.string.error_title)
+            characterDetailActivityError.visibility = View.VISIBLE
+            characterDetailActivityError.text =
+                getString(R.string.error_message_text, message, CHARACTER_DETAILS)
+        }
     }
 
     companion object {
         private const val CHARACTER_ID = "characterId"
+        private const val CHARACTER_DETAILS = "Character details"
```

Comment:
you are using this in the getString on line 96, put this in the same string res

Reply:
Refactored to string resource